index.js

```diff
@@ -9,7 +9,7 @@ var temporalDefaultOptions = {
 
 var excludeAttributes = function(obj, attrsToExclude){
   // fancy way to exclude attributes
-  return _.omit(obj, _.partial(_.rearg(_.contains,0,2,1), attrsToExclude));
```

Comment:
Do you understand what was the purpose of the code you deleted? As of now, I don't and it seems important to understand the intent.

Reply:
There's a lot to unwind in that one line. I've never used `_.partial` or `_.rearg`, so I spent a good amount of time looking at the Lodash docs, and the understanding I gained from that is:

> The deleted code uses a modified version of `_.contains` to determine whether a property is included in the passed `obj`, and if the property is included, that property is omitted from the final return object.

---

Comparing the documentation from Lodash 3 to Lodash 4, it looks like [`_.omit`](https://lodash.com/docs/3.10.1#omit) was broken into two functions in Lodash 4: [`_.omit`](https://lodash.com/docs/4.17.11#omit) and [`_.omitBy`](https://lodash.com/docs/4.17.11#omitBy). **

In Lodash 3:
- [`_.omit`](https://lodash.com/docs/3.10.1#omit) accepted a string, an array of strings, or a function.

In Lodash 4:
- [`_.omit`](https://lodash.com/docs/4.17.11#omit) accepts a string or array of strings
- [`_.omitBy`](https://lodash.com/docs/4.17.11#omitBy) accepts a function.

Back in `sequelize-temporal`:

This function (`excludeAttributes`) is used in two places when generating the History Model. In both cases, `attrsToExclude` is an array of strings (`excludedAttributes` and `excludedNames`). 

When using `_.omit`, and passing in an array of strings, I don't believe it matters whether one of the strings matches a property or not. If a property doesn't exist for one of the strings in the array, there's nothing to omit (the property is already not there, so it won't be in the return object). If the property exists, Lodash removes it from the return object.

As a result, I'm not sure using the modified version of `_.contains` is necessary.

```js
var obj = { one: 1, two: 2, three: 3 }

_.omit(obj, [ 'four', 'two' ]);    // note that obj['four'] doesn't exist
//> { one: 1, three: 3 }
```

---

If you'd prefer to keep using a function passed to `_.omit` in this:
- For Lodash 4, the old call to `_.omit` should be replaced with `_.omitBy` (** see above)
- `_.contains` was an alias to `_.includes` in Lodash 3. The alias was removed in Lodash 4, so `_.includes` should be used now.
- Other changes may be required, I never tried `_.omitBy` here.